airbyte-integrations/connectors/source-paypal-transaction/unit_tests/test_source.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from unittest.mock import MagicMock
+
+from source_paypal_transaction.source import (
+    PayPalOauth2Authenticator,
+    SourcePaypalTransaction,
+    get_endpoint,
+)
+
+
+class TestAuthentication:
+
+    def test_init_token_authentication_init(self, prod_config):
+        authenticator_instance = PayPalOauth2Authenticator(prod_config)
```

Comment:
If we move `authenticator_instance` to fixtures, that's looks better. 

```
@pytest.fixture()
def authenticator_instance(prod_config):
   return PayPalOauth2Authenticator(prod_config)
```

Also the same for `SourcePaypalTransaction()`.

wdyt?